crates/containerd-shim-wasm/Cargo.toml

```diff
@@ -53,3 +53,21 @@ systemd = ["libcontainer/systemd"]
 cgroupsv2 = ["libcontainer/v2"]
 cgroupsv1 = ["libcontainer/v1"]
 cgroupsv2_devices = ["libcontainer/cgroupsv2_devices"]
+
+[package.metadata.cargo-all-features]
+# Features "foo" and "bar" are incompatible, so skip permutations including them
```

Comment:
copy pasta?

Reply:
maybe a note about why `["cgroupsv2", "cgroupsv2_devices"]` etc is a combination we want to skip?

Reply:
Ah yes that's a copy and paste issue

Reply:
When I ran with these two features on, I got the following build error

```bash
  Compiling containerd-shim-wasm v0.2.1 (/home/mossaka/developer/runwasi/crates/containerd-shim-wasm)
error[E0308]: mismatched types
    --> /home/mossaka/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libcgroups-0.1.0/src/v2/devices/bpf.rs:58:17
     |
52   |             bpf_prog_load(
     |             ------------- arguments to this function are incorrect
...
58   |                 &opts,
     |                 ^^^^^ types differ in mutability
     |
     = note: expected raw pointer `*mut bpf_prog_load_opts`
                  found reference `&bpf_prog_load_opts`
note: function defined here
    --> /home/mossaka/.cargo/registry/src/index.crates.io-6f17d22bba15001f/libbpf-sys-1.2.1+v1.2.0/src/bindings.rs:4427:12
     |
4427 |     pub fn bpf_prog_load(
     |            ^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `libcgroups` (lib) due to previous error
```

Reply:
@utam0k , I believe this needs to be fixed upstream.

Reply:
@Mossaka , I suggest you ignore the `cgroupsv2_devices` feature for now. It seems to be broken upstream, and IIUC it's a no-op without enabling `cgroupsv2`.

Reply:
Removed `cgroupsv2_devices` feature

Reply:
@jprendes Thanks for your report 👀 
https://github.com/containers/youki/issues/new

Reply:
@jprendes @Mossaka I can't reproduce this error on my local. May I ask you to write how to reproduce this issue?
https://github.com/containers/youki/issues/2305

Reply:
I haven't tried reproduceding this locally, but just a thought. Are you using libcontainer from the `0.1` tag?

Reply:
hmm... I gave it a try to build it with `v0.1.0`, but I didn't get the error 🤔

Reply:
Hey, I cloned this PR added the `cgroupsv2_device` back in, and ran the `cargo build-all-features` and it completed without error as well